lms/static/scripts/frontend_apps/components/BookPicker.tsx

```diff
@@ -120,12 +122,12 @@ export default function BookPicker({
         />
       )}
       {step === 'select-chapter' && !error && (
-        <ChapterList
-          chapters={chapterList || []}
-          isLoading={!chapterList}
-          selectedChapter={chapter}
-          onSelectChapter={setChapter}
-          onUseChapter={confirmChapter}
+        <TableOfContentsPicker
+          entries={tableOfContents || []}
+          isLoading={!tableOfContents}
+          selectedEntry={chapter}
```

Comment:
There are some remaining uses of the term "chapter" left after this PR, which I can update later.